source/core/src/main/com/csse3200/game/ui/DialogueBox/DialogueBox.java

```diff
@@ -132,79 +170,71 @@ private Label createLabel() {
-        TextButton forwardButton = new TextButton("Continue", buttonStyle);
-        forwardButton.getLabel().setAlignment(Align.center);
+    private void createForwardButton() {
+        float desiredHeight = screenHeight * 0.05f;  // button image height
+        float newWidth = screenWidth * 0.10f;  // button image width
 
-        float buttonWidth = forwardButton.getWidth();
-        float centerX = (screenWidth - (2 * buttonWidth + 35)) / 2; // 35 is spacing between buttons
-
-        forwardButton.setPosition(centerX + buttonWidth + 20, label.getY() - 100);
+        TextButton.TextButtonStyle buttonStyle = createButtonStyle();
+        forwardButton = new TextButton("Continue", buttonStyle);
 
-        return forwardButton;
+        forwardButton.setSize(newWidth, desiredHeight);
+        forwardButton.setPosition(screenWidth / 2 + screenWidth * 0.1f, screenHeight * 0.03f);
+        forwardButton.getLabel().setAlignment(Align.center);
     }
 
     /**
      * Creates the backward button for navigating to the previous hint.
      *
      * @return The TextButton instance.
      */
-    private TextButton createBackwardButton() {
-        TextButton.TextButtonStyle buttonStyle = createButtonStyle();
-        TextButton backwardButton = new TextButton("Back", buttonStyle);
-        backwardButton.getLabel().setAlignment(Align.center);
-
-        float buttonWidth = backwardButton.getWidth();
-        float centerX = (screenWidth - (2 * buttonWidth + 35)) / 2; // 35 is spacing between buttons
+    private void createBackwardButton() {
+        float desiredHeight = screenHeight * 0.05f;  // button image height
+        float newWidth = screenWidth * 0.10f;  // button image width
 
-        backwardButton.setPosition(centerX, label.getY() - 100);
+        TextButton.TextButtonStyle buttonStyle = createButtonStyle();
+        backwardButton = new TextButton("Back", buttonStyle);
 
-        return backwardButton;
+        backwardButton.setSize(newWidth, desiredHeight);
+        backwardButton.setPosition(screenWidth / 2 - screenWidth * 0.1f - backwardButton.getPrefWidth(), screenHeight * 0.03f);
+        backwardButton.getLabel().setAlignment(Align.center);
     }
 
     /**
      * Creates the playButton for booting up mini-games
      *
      * @return the playButton instance.
      */
-    public TextButton createPlayButton() {
+    public void createPlayButton() {
         TextButton.TextButtonStyle buttonStyle = createButtonStyle();
-        TextButton playButton = new TextButton("Play Game", buttonStyle);
+        playButton = new TextButton("Play Game", buttonStyle);
         playButton.getLabel().setAlignment(Align.center);
         float buttonWidth = playButton.getWidth();
         float buttonHeight = playButton.getHeight();
         float centerX = (screenWidth - buttonWidth) / 2;
         float centerY = (screenHeight - buttonHeight) / 2;
 
         playButton.setPosition(centerX, centerY - 300);
```

Comment:
This my fault with the -300 (I'll make sure to fix that). But the screenWidth and screenHeight don't seem to update when the screen resizes.

Reply:
Now dynamic.